docs/getting-started.md

```diff
@@ -280,6 +282,9 @@ names of their fields are the same that are used in the API requests.
 An example of configuration file is provided:
 [`tests/framework/vm_config.json`](../tests/framework/vm_config.json).
 
+Once the guest is booted, refer [network-setup](./network-setup.md#in-the-guest)
```

Comment:
nit: A better place for this would be before line 260 immediately below the snippet to run the commands.

Reply:
But, isnt the network setup of guest common for both cases: creating VM via the API requests or via the config file.

Reply:
@sudanl0 Can you confirm on this ?

Reply:
sorry I missed your comment. Yes its common, I just thought it would be more readable few lines above when someone is just trying to copy paste stuff :) But its subjective so I am ok with the current change.